hypha/apply/funds/models/submissions.py

```diff
@@ -474,6 +478,31 @@ def restart_stage(self, **kwargs):
             return 'invited_to_proposal'
         return INITIAL_STATE
 
+    @transition(
```

Comment:
The state of the submission should not change when the determination is edited. My understanding is that the edits are only for archival purposes.